btcjson/btcdextcmds_test.go

```diff
@@ -193,7 +193,7 @@ func TestBtcdExtCmds(t *testing.T) {
 	for i, test := range tests {
 		// Marshal the command as created by the new static command
 		// creation function.
-		marshalled, err := btcjson.MarshalCmd(testID, test.staticCmd())
+		marshalled, err := btcjson.MarshalCmd("1.0", testID, test.staticCmd())
```

Comment:
Why are you changing from a variable to a hardcoded number here (and all the places below?

Reply:
fair point, I'll have to refactor. I've held off on squashing till these revisions are complete, but can do this earlier fi it helps your review

Reply:
Appreciate the feedback

Reply:
Generally I agree about waiting to squash until revisions are done but this needs the rebase before reviewing it for real so you might as well squash at the same time (I hate making people do extra steps if I don't have to).

Reply:
No problem